isis/src/base/objs/ProcessImport/ProcessImport.cpp

```diff
@@ -1297,6 +1297,7 @@ namespace Isis {
    */
   Isis::Cube *ProcessImport::SetOutputCube(const QString &fname,
       Isis::CubeAttributeOutput &att) {
+    SetAttributes(att);
```

Comment:
I  originally left this change out of this function because I didn't want to alter this function's behavior. I thought I moved the set attribute call(s) to `leisa2isis` where necessary. Is something still failing?

Reply:
leisa2isis was failing without it. Jenkins now has some kaguya2isis errors ad I'll see if it's related.

Reply:
undoing this didn't fix it? I think it's someone messing with Kaguya data.

Reply:
Also failing on dev so not related.